share/spice/xkcd/display/xkcd_display.js

```diff
@@ -29,31 +66,41 @@
                     sourceName: 'xkcd',
                     sourceIcon: true
                 },
+                normalize: function(item) {
+                    var subtitles = [];
+                    
+                    subtitles.push(build_comic_number_and_created_date_string(item.num, item.month, item.day, item.year));
+                    
+                    if (item.num > 1) {
+                        subtitles.push({ href: "/?q=xkcd+"+(item.num-1), text: "Previous" });
+                    }
+                    
+                    if (item.has_next) {
+                        subtitles.push({ href: "/?q=xkcd+"+(item.num+1), text: "Next" });
+                    }
+                    
+                    subtitles.push({ href: "/?q=xkcd+"+item.random_xkcd_number, text: "Random" });
+                    
+                    subtitles.push({ href: "http://www.explainxkcd.com/wiki/index.php/"+item.num, text: "Explain it!" });
+                    
+                    return {
+                        title: item.title,
+                        subtitle: subtitles,
+                    }
+                },
                 templates: {
-                    group: 'base',
+                    group: 'text',
                     options: {
                         content: Spice.xkcd_display.content,
                         moreAt: true
                     }
                 }
             });
+            
+            if (is_mobile) {
+                $('.c-base__sub .sep').first().replaceWith("<br>");
```

Comment:
This needs to be targetted to the IA's `<div>` -- otherwise this will change the first seperator on the page which isn't guaranteed to be the one in the IA.

You need something like `(.zci--xkcd .c-base__sub .sep')`